### PR TITLE
Completed all tests for JavaScript Constructors.

### PR DESCRIPTION
**Pull request by owen-adams2021**
image
image
image
image

### CODE REVIEW

File: javascript-constructors/new-objects.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+console.log('value of anArray:', anArray);
+console.log('typeof anArray:', typeof anArray);
+
+var aFunction = new Function();
+console.log('value of aFunction:', aFunction);
+console.log('typeof aFunction:', typeof aFunction);
+
+var aString = new String();
+console.log('value of aString:', aString);
+console.log('typeof aString:', typeof aString);
```
Comment (codingcodymiller) [inline javascript-constructors/new-objects.js:15]: You seem to be missing a few here 🤔 
Screen Shot 2021-02-16 at 2 34 32 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-constructors/new-objects.js (hunk 1 above)
Comment (owen-adams2021) [reply javascript-constructors/new-objects.js:15]: Big oof! Will fix :)

**Comment by owen-adams2021** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
🎉

**Pull request merged**